src/pages/ProductLists/ProductLists.js

```diff
@@ -1,10 +1,103 @@
 import React, { Component } from 'react';
+import ProductContainer from '../../component/ProductContainer/ProductContainer';
+import Modal from '../../component/Modal/Modal';
+import '../ProductLists/ProductLists.scss';
+
+class ProductLists extends Component {
+  constructor() {
+    super();
+    this.state = {
+      product: [],
+      modalInfo: [],
+      isModalOn: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/sandBag.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(info => {
+        this.setState({
+          product: info,
+        });
+      });
+
+    fetch('http://localhost:3000/data/modalData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(sequence => {
+        this.setState({
+          modalInfo: sequence,
+        });
+      });
+  }
+
+  // filter 함수 구현 시작
+
+  searchName = () => {
+    const { product } = this.state;
+    return product.filter(el => {
+      return el.discountPrice >= 300000;
+    });
+  };
+
+  //모달창 on/off 함수 구현 시작
+
+  handleClick = () => {
+    this.setState(state => ({
+      isModalOn: !state.isModalOn,
+    }));
+  };
 
-export class ProductLists extends Component {
   render() {
+    const { product, modalInfo, isModalOn } = this.state;
+    console.log(modalInfo);
     return (
-      <div>
-        <h1>ProductLists</h1>
+      <div className="Container">
+        <div className="categoryTitle">
+          <p>침대</p>
+
+          <div className="checkBox">
+            <button className="filterBtn" onClick={this.handleClick}>
+              인기순 ▼{isModalOn === true ? 'optionBox' : null}
```

Comment:
=== true 부분은 불필요한 것 같네요 그리고 true일 경우만 처리하려면 삼항연산자보다는 &&연산자가 간결합니다!
```suggestion
              인기순 ▼{isModalOn && 'optionBox'}
```

Reply:
&& 연산자를 사용하는 것에 있어 완전히 이해하지 못해
사용하지 못했는데 앞으로는 true일 경우만 처리할 땐 && 연산자를
애용해보도록 하겠습니다!

Reply:
&& || 연산자 단축평가 라는 키워드로 찾아보시면 어떤식으로 동작하는 건줄 알 수 있을 겁니다!!